src/sagas/actionSaga.ts

```diff
@@ -0,0 +1,31 @@
+import axios from 'axios';
+import { Action as ReduxAction } from 'redux-actions';
+import { call, put, takeLatest } from 'redux-saga/effects';
+
+import { updateLastActiveAt } from 'reducers/authReducer';
+import { Action } from 'types/action';
+import { postSystemIntakeAction } from 'types/routines';
+
+function postSystemIntakeActionRequest(formData: Action) {
+  return axios.post(
+    `${process.env.REACT_APP_API_ADDRESS}/system_intake/${formData.intakeId}/actions`,
+    `"${formData.actionType}"`
```

Comment:
Is it dangerous that the request body is just a single string? 
This way feels a bit hacky with inserting quotes around it so it still processes under content-type: json. We could change the heading here to Content-Type: text but I'm not sure if that opens security vulnerabilities for us. Would love to hear input on this

Reply:
We could put it in an object. This might be better in case we need to add additional keys in the body.
```
{
  type: formData.actionType
}
```